ui/app/AppLayouts/Onboarding2/UseRecoveryPhraseFlow.qml

```diff
@@ -18,15 +18,15 @@ SQUtils.QObject {
     signal setPasswordRequested(string password)
     signal finished
 
-    function init() {
-        root.stackView.push(seedphrasePage)
+    function init(flow: int) {
```

Comment:
I was trying to avoid passing that enum into subflows and use it only one layer above. This flow should be unaware of all those things that this enum describes.

Maybe we could define local enum for that component with "type" of usage, e.g. "LoggingIn" and "ProfileCreation"?

Reply:
Done